reactivesocket/src/test/scala/akka/stream/alpakka/reactivesocket/scaladsl/ReactiveSocketSpec.scala

```diff
@@ -0,0 +1,94 @@
+package akka.stream.alpakka.reactivesocket.scaladsl
+
+import akka.stream.scaladsl._
+import io.reactivesocket.util.PayloadImpl
+import org.scalatest.WordSpec
+
+import scala.concurrent.duration._
+
+class ReactiveSocketSpec extends WordSpec {
+
+  "reactive socket API" should {
+
+    "support server and client" in {
+
+      import io.reactivesocket.transport.tcp.client.TcpTransportClient
+      import io.reactivesocket.transport.tcp.server.TcpTransportServer
+
+      val binding = ReactiveSocketServer(TcpTransportServer.create())
+          .toMat(Sink.foreach {
+            _.flow.runWith(Source.repeat(new PayloadImpl("Pong")), Sink.foreach(println))
```

Comment:
How would the different interaction types (fire-forget, request-response, ...) be represented?
Compare with https://github.com/akka/alpakka/pull/26/files#diff-35147bb7a6ac0eb12c60538ecb260128R45